### PR TITLE
remove deprecated fast liquidity path

### PR DESCRIPTION
**Pull request by anna-carroll**
## Motivation
Fast liquidity was deprecated as part of code cleaning related to the audit process. We removed the ability to send new fast liquidity messages, and the ability to pre-fill fast liquidity transfers. However, we kept the code related to handling fast liquidity messages, in case of any in-flight messages. Now, we can remove the ability to handle these messages to deprecate the feature entirely.

## Solution
Remove the ability to handle fast liquidity messages to simplify the code.

## PR Checklist

- [ ] Added Tests
- [ ] Updated Documentation
- [ ] Updated CHANGELOG.md for the appropriate package

### CODE REVIEW

**Review by odyslam — APPROVED** [review]
We assume now that all in-flight messages have been transmitted?

File: packages/contracts-bridge/contracts/BridgeMessage.sol [hunk 1]
```diff
@@ -20,7 +20,7 @@ library BridgeMessage {
         TokenId, // 1
         Message, // 2
         Transfer, // 3
-        FastTransfer // 4
```
[the remark this answers is not in the document]
Comment (prestwich) [reply packages/contracts-bridge/contracts/BridgeMessage.sol:23]: if we never repurpose a number we can't accidetnally introduce type confusion in the future

**Review by prestwich — APPROVED** [review]

**Review by prestwich — APPROVED** [review]

**Comment by anna-carroll** [comment]
Note: before we ship this upgrade, we will have to run a script ensuring all "fast liquidity" message (if there ever were any) have been delivered

**Comment by prestwich** [comment]
@kekonen should be able to make a quick indexer query for this?

**Pull request merged**